ui/desktop/app/templates/scopes/scope/projects/sessions/index.hbs

```diff
@@ -201,11 +201,12 @@
               <SessionStatus @model={{B.data}} />
             </B.Td>
             <B.Td>
-              {{#if B.data.isAvailable}}
+              {{#if (can 'cancel session' B.data)}}
```

Comment:
None blocking comment: Didn't check with design perhaps this is their desired behaviour. But I am thinking, is a user has no grants to cancel a session, shouldn't we disable the button instead of hidding it?

Reply:
I had seen a [recommendation](https://hashicorp.atlassian.net/wiki/spaces/DES/pages/2678685874/Hiding+Disabling) before that we should be hiding things users don't have permissions for rather than disabling so was just going off of that

Reply:
That makes sense!! Thanks for pointing out the recommendation!